tools/bamm-cli/src/main/java/io/openmanufacturing/sds/aspect/AspectValidateCommand.java

```diff
@@ -54,8 +61,10 @@ public void run() {
 
       final List<Violation> violations = validator.validateModel( versionedModel );
       if ( details ) {
+         LOG.debug( "Printing detailed validation results" );
```

Comment:
Do we really need this? Doesn't the output formats differ enough to easily recognize which version is being printed?

Reply:
You are right, it's not really necessary.